src/reflection.jl

```diff
@@ -39,17 +39,43 @@ function highlight(io::IO, code, lexer)
     return
 end
 
+#
+# Compat shims
+# 
+
+@inline function typed_signature(@nospecialize(job::CompilerJob))
+    u = Base.unwrap_unionall(job.source.tt)
+    return Base.rewrap_unionall(Tuple{job.source.f, u.parameters...}, job.source.tt)
+end
+
+function method_instances(@nospecialize(tt::Tuple), world::UInt=Base.get_world_counter())
+    return map(Core.Compiler.specialize_method, method_matches(tt; world))
+end
 
 #
 # code_* replacements
 #
 
-code_lowered(@nospecialize(job::CompilerJob); kwargs...) =
-    InteractiveUtils.code_lowered(job.source.f, job.source.tt; kwargs...)
+function code_lowered(@nospecialize(job::CompilerJob); generated::Bool=true, debuginfo::Symbol=:default)
```

Comment:
Ah, I had hoped we didn't have to replicate the entire definition...

Reply:
Yeah didn't see a way around it

Reply:
Add a note then, we preferably want to extend InteractiveUtils not to need this def anymore.